src/index.html

```diff
@@ -3,11 +3,296 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/favicon.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header id="home" class="header">
+      <a href="#home" class="header__logo">
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="Bose logo"
+        >
+      </a>
+
+      <div class="header__action">
+        <a class="header__phone phone" href="tel:+1 234 555-55-55">
+          <img
+            class="phone__icon"
+            src="images/phone.svg"
+            alt="Phone icon"
+          >
+        </a>
+
+        <span class="phone__number">+1 234 555-55-55</span>
+
+        <a class="header__menu" href="#menu">
+          <img
+            class="header__menu-icon"
+            src="images/menu.svg"
+            alt="Menu icon"
+          >
+        </a>
+      </div>
+
+      <div class="header__content">
+        <h1 class="header__motto">
+          The world shades.
+          <br>
+          Your music shines.
+        </h1>
+      </div>
+    </header>
+
+    <nav id="menu" class="menu">
```

Comment:
This navigation is part of the header, I would move it there

Reply:
I moved navigation to header.